### ISSUE TITLE
Test output does not give file in which test failed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [typetools/checker-framework#26]
```
Try running ant on the Checker Framework tests.  The output doesn't tell
you what test failed, only the line number at which it failed.

Here is an example, when running "ant nullness-tests":

nullness-tests:

-run-tests:
Running tests.NullnessTest
2 out of 2 expected diagnostics were found.

1 unexpected diagnostic was found:
:28: (type.incompatible)
)
Tests run: 46, Failures: 1, Errors: 1, Time elapsed: 19.078 sec

BUILD FAILED

```

Original issue reported on code.google.com by `michael.ernst@gmail.com` on 23 Nov 2009 at 11:50
